src/platform/Darwin/DnssdImpl.h

```diff
@@ -168,6 +168,7 @@ struct BrowseContext : public BrowseHandler
 {
     DnssdBrowseCallback callback;
     std::vector<DnssdService> services;
+    std::vector<std::pair<DnssdService, std::string>> domainNames;
```

Comment:
This doesn't make sense.  We already have a vector of DnssdService.   Why are we having another vector of the same DnssdService?

What we talked about was having `services` be a vector of either std::pair<DnssdService, std::string>, or a vector of some struct that has DnssdService and std::string members, or a vector of some struct that inherits from DnssdService and has the std::string member in addition to that, no?

Or having just a vector of std::string that matches the vector of DnssdService, if we are creating Span<DnssdService> from the `services` vector....

Reply:
I fixed this already
. Have one vector of std::vector<std::pair<DnssdService, std::string>> services.